### PR TITLE
[Rsdk-627] audit and mark rdk packages experimental

### PR DESCRIPTION
**Pull request by randhid**

### CODE REVIEW

**Review by npmenard — APPROVED** [review]
LGTM! do we have to comment each files within the same package?

File: components/movementsensor/gpsrtk/gpsrtk.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Package gpsrtk defines a gps and an rtk correction source
 // which sends rtcm data to a child gps
-// Experimental package
+// This is an Experimental package
```
Comment (stevebriskin) [inline components/movementsensor/gpsrtk/gpsrtk.go:3]: i missed this...why did we decide that rtk isn't beta-ready?

File: components/movementsensor/cameramono/cameramono_test.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// This is an Experimental package
```
Comment (stevebriskin) [inline components/movementsensor/cameramono/cameramono_test.go:1]: just the test file?

**Review by stevebriskin — APPROVED** [review]
lgtm, minus the cameramono comment.

File: components/movementsensor/gpsrtk/gpsrtk.go (hunk 1 above)
Comment (randhid) [reply components/movementsensor/gpsrtk/gpsrtk.go:3]: I want to keep the door open for some config changes.

We have an 'rtk_station' concept that doesn't show anything and just sends data to child gps. Some of the station's functionality is repeated in the regular 'gps-rtk' that connects a specific chip and uses the internet to get correction data from there. There's probably a better and less user-heavy way of configuring these two models.

**Pull request merged**